lib/dailymotion-api/client.rb

```diff
@@ -46,31 +55,35 @@ def get_video(video_id, fields = "")
       return nil unless video_id
 
       url = "#{API_URL}/video/#{video_id}"
-      url << "?fields=#{fields}" unless fields.empty?
+      url += "?fields=#{fields}" unless fields.empty?
 
       response = HTTMultiParty.get(url)
       response.parsed_response
     end
 
     def video_url
-      @video_url ||= get_video(@video_id, "url")["url"] rescue nil
+      @video_url ||=  begin
+                        get_video(@video_id, "url")["url"]
+                      rescue StandardError
+                        nil
+                      end
```

Comment:
Because you hide all errors and don't catch only what you expect. I had no time to think about what errors you expect here. So, I just fixed Rubocop offence on this line.